jiant/proj/main/modeling/model_setup.py

```diff
@@ -340,6 +357,16 @@ def get_model_arch_from_jiant_model(jiant_model: nn.Module) -> ModelArchitecture
 
 
 def get_ancestor_model(transformers_class_spec, model_config_path):
+    """Load the model config from a file, configure the model, and return the model.
```

Comment:
Clarify that the model here, in the case of Transformer models, corresponds to the model class that contains all of the pretrained weights (e.g. both MLM and NSP heads)

Reply:
Thanks, done w/ 64b5a99.